### PR TITLE
Docs on consent bits

### PR DESCRIPTION
**Pull request by richvdh**

### CODE REVIEW

File: docs/consent_tracking.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+---------------------------------------
+
+Synapse can be configured to serve the user a simple policy form with an
+"accept" button. Clicking "Accept" records the user's acceptance in the
+database and shows a success page.
+
+To enable this, first create templates for the policy and success pages.
+These should be stored on the local filesystem.
+
+These templates use the [Jinja2](http://jinja.pocoo.org) templating language,
```
Comment (neilisfragile) [inline docs/consent_tracking.md]: I found it hard to parse this section, perhaps 
"These templates use the Jinja2 templating language, and `docs/privacy_policy_templates` gives examples of the sort of thing that can be done.''

File: docs/consent_tracking.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+      - port: 8008
+        resources:
+          - names:
+            - client
+            - consent
+    ```
+
+
+Finally, ensure that `jinja2` is installed. If you are using a virtualenv, this
+should be a matter of `pip install Jinja2`. On debian, try `apt-get install
```
Comment (neilisfragile) [inline docs/consent_tracking.md]: Why is jinja2 a special cased dependency?

File: docs/consent_tracking.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+python-jinja2`.
+
+Once this is complete, and the server has been restarted, try visiting
+`https://<server>/_matrix/consent`. If configuration has been done correctly,
```
Comment (neilisfragile) [inline docs/consent_tracking.md]: maybe "If correctly configured this should give an error...."

File: docs/server_notices.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+Server Notices
+==============
+
+'Server Notices' are a new feature introduced in Synapse 0.30. They provide a
+channel whereby server administrators can send messages to users on the server.
+
+They are used as part of the communication of the server polices (see
```
Comment (neilisfragile) [inline docs/server_notices.md]: drop the 'the'?

File: docs/server_notices.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+The user is prevented from sending any messages in this room by the power
+levels. They also cannot leave it.
+
+Synapse configuration
+---------------------
+
+Server notices come from a specific user id on the server. Server
+administrators are free to choose the user id - something like `server` is
+suggested, meaning the notices will come from
+`@server:<your_server_name>`. Once the server notices user is configured, that
```
Comment (neilisfragile) [inline docs/server_notices.md]: Easy to misinterpret as the server noticing something - at first I thought it was a typo. Perhaps capitalise Server Notices User?

File: docs/consent_tracking.md (hunk 2 above)
Comment (richvdh) [reply docs/consent_tracking.md]: well, because if you aren't doing this sort of stuff, you don't need it. Though maybe we should reconsider this.

File: docs/consent_tracking.md (hunk 1 above)
Comment (richvdh) [reply docs/consent_tracking.md]: I suspect it's not in `docs` if you've got a debian installation :/

File: docs/server_notices.md (hunk 4 above)
Comment (richvdh) [reply docs/server_notices.md]: "part of communication of the server polices" ? really?

"part of communicating" maybe?

File: docs/server_notices.md (hunk 5 above)
Comment (richvdh) [reply docs/server_notices.md]: good point, done

File: docs/consent_tracking.md (hunk 1 above)
Comment (richvdh) [reply docs/consent_tracking.md]: ok, should be better now
